src/homeworks/homework18/bank/BankTest.java

```diff
@@ -0,0 +1,40 @@
+package homeworks.homework18.bank;
+
+import java.util.Arrays;
+
+public class BankTest {
+    public static void main(String[] args) {
+        User userOne = new User("Petro", 30);
+        Account debitAccount = new Account("100", 1000);
+        Account creditAccount = new Account(101, 1001);
```

Comment:
Maybe it's better to create some separate classes ?
```suggestion
        DebitAccount debitAccount = new DebitAccount("100", 1000);
        CreditAccount creditAccount = new CreditAccount(101, 1001);
```